### PR TITLE
[KOGITO-6998] Adding alias configuration option

### PR DESCRIPTION
**Pull request by fjtirado**
**JIRA:**

**Description:**

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

### CODE REVIEW

**Review by spolti — APPROVED** [review]

**Review by hbelmiro — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc [hunk 1]
```diff
@@ -113,6 +113,41 @@ In the previous example, Kogito uses `stock-service_API` as configuration key.
 A Kubernetes service endpoint can be used as a service URL if the target service is within the same cluster, such as `http://myservice.mynamespace.cluster.svc.local`.
 ====
 
+=== Using URI alias
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc:116]:
```suggestion

```

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc [hunk 2]
```diff
@@ -113,6 +113,41 @@ In the previous example, Kogito uses `stock-service_API` as configuration key.
...
+
+As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as configuration key and in function definitions.
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc:118]:
```suggestion
As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as a configuration key in function definitions. 
```

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc [hunk 3]
```diff
@@ -113,6 +113,41 @@ In the previous example, Kogito uses `stock-service_API` as configuration key.
 A Kubernetes service endpoint can be used as a service URL if the target service is within the same cluster, such as `http://myservice.mynamespace.cluster.svc.local`.
 ====
 
+=== Using URI alias
+
+As an alternative to `kogito.sw.operationIdStrategy`, you can assign an alias name to an URI by using `workflow-uri-definitions` custom link:{spec_doc_url}#extensions[extension]. Then you can use that alias as configuration key and in function definitions. 
+
+.Example Serverless Workflow
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc:120]:
```suggestion
.Example workflow
```

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc [hunk 4]
```diff
@@ -113,6 +113,41 @@ In the previous example, Kogito uses `stock-service_API` as configuration key.
...
+    ]
+----
+
+.Example property
+[source, properties]
+----
+quarkus.rest-client.remoteCatalog.url=http://localhost:8282/
+----
+
+In the previous example, Kogito uses `remoteCatalog` as configuration key.
```
Comment (hmanwani-rh) [inline serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc:148]:
```suggestion
In the previous example, {product_name} uses `remoteCatalog` as configuration key.
```

**Review by hmanwani-rh — COMMENTED** [review]
@fjtirado I've added a few suggestions below. Couple of suggestions are based on the product name changes that we recently did in the guides. 

Please take a look and let me know if you've any questions.

File: serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc (hunk 2 above)
Comment (fjtirado) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc:118]: This change the meaning, I meant you can use the alias as configuration AND also use the alias in function definitions

Comment (hmanwani-rh) [reply serverlessworkflow/modules/ROOT/pages/service-orchestration/configuring-openapi-services-endpoints.adoc:118]: Ah, okay. Done, I've updated the suggestion

**Pull request merged**